CRM/Upgrade/Incremental/sql/5.15.alpha1.mysql.tpl

```diff
@@ -9,3 +9,33 @@ SELECT @option_group_id_ps as option_group_id, {localize field='label'}`label`{/
 FROM civicrm_option_value ov
 INNER JOIN civicrm_option_group og
 ON og.id = ov.option_group_id AND og.name = 'contribution_status';
+
+SELECT @maxValue := MAX(value) FROM `civicrm_option_value` where option_group_id = @option_group_id_ps;
```

Comment:
value is of type VARCHAR so MAX won't be accurate unless we convert it. 

`SELECT @maxValue := MAX(CAST( `value` AS UNSIGNED )) FROM `civicrm_option_value` where option_group_id = @ option_group_id_ps;`